featuretools/tests/primitive_tests/utils.py

```diff
@@ -0,0 +1,5 @@
+def get_number_of_days(offset):
```

Comment:
Probably not a big deal since we are only using this in tests, but I think this would return the wrong value if we ever used this in a test with a number above 9.

Reply:
Yeah, I can either clarify the limitations in a docstring/what the intended use is or update it to return any numeric characters before a string. I think my preference for now would be to keep it simple and add a docstring.

Reply:
Still need to add the docstring here?

Reply:
oops, yeah forgot to push